op-chain-ops/genesis/config.go

```diff
@@ -356,7 +362,13 @@ func (d *DeployConfig) Check() error {
 		log.Warn("GasPriceOracleOverhead is 0")
 	}
 	if d.GasPriceOracleScalar == 0 {
-		return fmt.Errorf("%w: GasPriceOracleScalar cannot be 0", ErrInvalidDeployConfig)
+		log.Warn("GasPriceOracleScalar is 0")
+	}
+	if d.GasPriceOracleBaseFeeScalar == 0 {
+		log.Warn("GasPriceOracleBaseFeeScalar is 0")
+	}
+	if d.GasPriceOracleBlobBaseFeeScalar == 0 {
+		log.Warn("GasPriceOracleBlobBaseFeeScalar is 0")
 	}
```

Comment:
We don't yet support ecotone at genesis so it would just be extra branching logic. I lean towards not being too strict here for simplicity, it would also break old configs that do not yet have the new fields. We need to think of the entire UX of chain deployments when it comes to the deploy config changes, there is a lot of tech debt here